docs/source/gui.rst

```diff
@@ -1,100 +1,149 @@
 GUI
 ===
 
-Using the Data Browser
-----------------------
+Using the GUI
+-------------
 
-**Note**: the data browser requires ``ipywidgets`` to be available in the current environment
+**Note**: the GUI requires the latest release versions of ``panel`` and ``bokeh`` to
+be available in the current environment.
 
 The Intake top-level singleton ``intake.gui`` gives access to a graphical data browser
 within the Jupyter notebook. To expose it, simply enter it into a code cell (Jupyter
 automatically display the last object in a code cell).
 
-.. image:: _static/images/gui1.png
+.. image:: _static/images/gui_builtin.png
 
-New instances of the GUI are also available by instantiating ``intake.DataBrowser``,
-where you can specify a set of catalogs to initially include.
+New instances of the GUI are also available by instantiating ``intake.GUI``,
+where you can specify a list of catalogs to initially include.
 
+The GUI contains three main areas:
 
-The GUI contains:
-
-- a list of catalogs. The "builtin" catalog, displayed by default, includes data-sets installed
+- a **list of catalogs**. The "builtin" catalog, displayed by default, includes data-sets installed
   in the system, the same as ``intake.cat``.
 
-- a list of entries within the currently-selected catalog. Entries marked with ``" ->"``
-  are themselves catalogs, clicking them will load the catalog and update the left-hand
-  listing. Clicking any other entry will display some basic information in the right-hand text
-  box.
+- a **list of sources** within the currently-selected catalog.
+
+- a **description** of the currently-selected source.
+
 
-- The following controls:
+Catalogs
+--------
+Selecting a catalog from the list will display nested catalogs below the parent and display
+source entries from the catalog in the **list of sources**.
 
-  -  Search: opens a sub-panel for finding entries in the currently-selected catalog (and it's
-     sub-catalogs)
+Below the **lists of catalogs** is a row of buttons that are used for adding, removing and
+searching-within catalogs:
 
-  -  Add: opens a sub-panden for adding catalogs to the interface, by either browsing for a local
-     YAML file or by entering a URL for a catalog, which can be a remote file or Intake server
+-  **Search**: opens a sub-panel for finding entries in the currently-selected catalog (and it's
```

Comment:
Should be in the order in the UI (which I think is the right order): add, remove, search. The sections below will need to be shuffled too.

Reply:
Do you think the section titles should just be "Add", "Remove", and "Search"?